### DISCUSSION TITLE
Doesn't install on Arch Linux: sqlite2-2.8.17-9 build error

### DISCUSSION THREAD

**Discussion by mireiner** [pawelsalawa/sqlitestudio#4128] [General]
### Details
sqlite2-2.8.17-9 build fails with error message:
./src/shell.c:695:20: error: format not a string literal and no format arguments [-Werror=format-security]
  695 |     fprintf(stderr,zHelp);
cc1: some warnings being treated as errors
make: *** [Makefile:199: sqlite] Error 1

### Steps to reproduce
Installing sqlitestudio using pamac gui application
or using aur helpers:
yay -S sqlitestudio
or
paru -S sqlitestudio

### Operating system
Arch Linux (vanilla)

### SQLiteStudio version
SQLiteStudio 3.3.3

**Comment by pawelsalawa** [comment]
I have no idea about Arch packages. You need to reach out to the maintainer of this package on Arch.
